package.json

```diff
@@ -40,12 +40,15 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-import-resolver-typescript": "^3.5.1",
     "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-no-async-foreach": "^0.1.1",
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-react": "^7.30.1",
     "growthbook": "^0.2.1",
     "husky": "^7.0.0",
     "lint-staged": "^10.2.7",
+    "patch-package": "^8.0.0",
```

Comment:
We need to use `patch-package` to get around a blocking TypeScript error caused by an issue in the Octokit library. There is a currently open PR to address the issue, [here](https://github.com/octokit/webhooks.js/pull/949).